.github/workflows/cf-deploy.yml

```
@@ -28,6 +28,9 @@ jobs:
           elif [ "${{ github.ref }}" = "refs/heads/master" ]; then
             echo "::set-output name=env_name::staging"
             echo "::set-output name=build_script::npm run build-staging"
+          elif [ "${{ github.ref }}" = "refs/heads/gp4btc-staging" ]; then
+            echo "::set-output name=env_name::staging"
+            echo "::set-output name=build_script::npm run build-staging"            
```

I think we need a new `npm` command to execute the build with the `gp4btc-staging` values: https://energyweb.atlassian.net/browse/DOI-2417
```suggestion
            echo "::set-output name=build_script::npm run build-gp4btc-staging"            
```

---

@jrhender Will we have different backend for that staging env?

---

Are we planning to have more instances for gp4btc? For example dev/staging/prod? If not i will follow with what we already have in repo, so the script will be named `build-gp4btc`